.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-python: "3.4"
+python: "3.5"
```

Comment:
fhirclient has some circular imports that apparently break in 3.4 but don't in 3.5. I didn't look into why, I just upgraded my python.